packages/hash/api/src/index.ts

```diff
@@ -50,6 +51,9 @@ import { ensureSystemEntitiesExists } from "./graph/system-entities";
 const shutdown = new GracefulShutdown(logger, "SIGINT", "SIGTERM");
 
 const main = async () => {
+  await TypeSystemInitializer.initialize();
+  logger.info("Type System initialized");
```

Comment:
Do we want to merge this line?

Reply:
How would you want to do that?

Reply:
I was under the impression that this was just a `console.log` for local debugging. Happy with this line on `main` if it’s there on purpose.

Reply:
Gotcha, it's there on purpose :) There are other logger statements below, it's helpful to know how far the API got before it crashed during development